packages/generator/src/utils/modal-names.ts

```diff
@@ -0,0 +1,14 @@
+import {pluralCamel, pluralPascal, singleCamel, singlePascal} from "../index"
+
+export function modelName(input: string = "") {
```

Comment:
plz fix this file name from 'modal-names' to 'model-names' or better yet imo 'model-name-utils'

Reply:
Yeah, I did not realize the typo, I think I will rename it to `model-names` as it is already present in the utils folder.